src/main/java/lotto/domain/WinningLotto.java

```diff
@@ -18,4 +24,21 @@ public int matchCount(Lotto myLotto){
 
         return matchCount;
     }
+
+    public Rank matchRank(Lotto lotto) {
+        int myMatchCount = matchCount(lotto);
```

Comment:
```suggestion
        int myMatchCount = winningLotto.match(matchCount(lotto));
```

메시지를 보내는 방식으로 구현할 수 있지 않을까?